assets/js/src/form_editor/store/reducers/toggle_sidebar_panel.jsx

```diff
@@ -21,11 +21,11 @@ const getRequiredAction = (openedPanels, panelId, toggleTo) => {
  * @return {object} Modified state object
  */
 export default (state, action) => {
-  if (action.toggleTo !== undefined && !['opened', 'closed'].includes(action.toggleTo)) {
-    throw new Error(`Unexpected toggleTo value "${action.toggleTo}"`);
-  }
+  let toggleTo;
```

Comment:
Just wondering, why this change had to be done?

Reply:
Because the component doesn't return string any more, it returns a boolean value.